.gitignore

```diff
@@ -28,3 +28,6 @@ ccdaservice/node_modules
 # testing
 .phpunit.result.cache
 coverage/js-unit
+
+#sites
+sites
```

Comment:
Will need to remove this change since can't ignore codebase stuff in this directory